src/streams/StreamSettingsScreen.js

```diff
@@ -60,6 +62,11 @@ class StreamSettingsScreen extends PureComponent<Props> {
     NavigationService.dispatch(navigateToStreamSubscribers(stream.stream_id));
   };
 
+  handleSubscribe = () => {
```

Comment:
How about a name that says that it's handling a request to add a subscription, to make it clear it's not for unsubscribing, or a toggle between them?

I have mixed feelings about the naming pattern that's developed for these methods we use as event handlers. On the one hand, I guess I can be glad that [most of them] say "handle", since that defends them against being haphazardly co-opted into use as something other than an event handler. At the same time, "ToggleMuteStream", "Edit", "EditSubscribers", etc., aren't the best descriptions for events that these things fire on. Rather, they fire on the user _requesting_ to toggle stream muting, to edit, to edit subscribers, etc., by doing the appropriate interaction. Possibly `handleRequestEdit` would be a reasonable name, but I'm not sure I'm up to enforcing that for anything much longer; `handleRequestToggleMuteStream` might obscure more than it clarifies. Do you have thoughts on this?

I think I'd be fine with keeping the general pattern but adding "add" in there somewhere, like `handleSubscriptionAdd` or something similar.

Reply:
How do you feel about `handleSubscribeClick`? `Click` is sort of anachronistic, but used a fair amount, so I don't think it'd be confusing.

Reply:
Yeah, probably fine, maybe better, to have it be explicit about the type of interaction the user uses to say they want to subscribe. Better because in cases where we actually use an event object passed to the handler, the event object will have details specific to the type of interaction.

Except my sense is that "press" is more idiomatic on mobile; `ZulipButton` and the things from RN that it uses have `onPress`. So, `handleSubscribePress` or `handlePressSubscribe` would be fine; I think I prefer the latter.

Reply:
Ok, changed to `handlePressFoo`. Did this as a commit at the end, since that felt cleanest (given that I was updating existing code as well)